cmd/auth-provider-gcp/apis/credentialprovider/v1alpha1/BUILD

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "doc.go",
+        "register.go",
+        "types.go",
+        "zz_generated.conversion.go",
+        "zz_generated.deepcopy.go",
+        "zz_generated.defaults.go",
+    ],
+    importmap = "k8s.io/kubernetes/vendor/k8s.io/kubelet/pkg/apis/credentialprovider/v1alpha1",
+    importpath = "k8s.io/kubelet/pkg/apis/credentialprovider/v1alpha1",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
```

Comment:
Don't think staging should work in this repo.

Reply:
Fixed (confused as to how it's been building/testing properly up to this point since I know `provider/` hangs a dependency on this directory...)